src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -464,21 +464,50 @@ private void testConstSumSubmitAction() {
         assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-"
                                                      + qnNumber + "-" + responseNumber));
 
-        // Test messages for different values entered
         submitPage = loginToInstructorFeedbackSubmitPage("IFSubmitUiT.instr", "Open Session");
+
+        // Test instructions displayed for filling the form
+        qnNumber = 18;
+        assertEquals("Total points distributed should add up to 400.",
+                     submitPage.getConstSumInstruction(qnNumber));
+
+        qnNumber = 19;
+        assertEquals("Total points distributed should add up to 400.\n"
+                     + "Every recipient should be allocated different number of points.",
+                     submitPage.getConstSumInstruction(qnNumber));
+
+        qnNumber = 20;
+        assertEquals("Total points distributed should add up to 100.\n"
+                    + "Every option should be allocated different number of points.",
+                    submitPage.getConstSumInstruction(qnNumber));
+
+        qnNumber = 21;
+        assertEquals("Total points distributed should add up to 400.\n"
+                    + "At least one recipient should be allocated different number of points.",
+                    submitPage.getConstSumInstruction(qnNumber));
+
+        qnNumber = 22;
+        assertEquals("Total points distributed should add up to 100.\n"
+                    + "At least one option should be allocated different number of points.",
+                    submitPage.getConstSumInstruction(qnNumber));
```

Comment:
do the above questions include const sum option questions?

Reply:
then how come the tests can pass for the const sum option questions if these instructions dont appear for those questions without recipient names?

Reply:
the tests are passing because almost all the questions have the recipient names (for options). I didn't consider any new questions. I augmented the tests for the existing const sum questions that were being tested as part of method `testConstSumSubmitAction`. Question 17 is the only question without the recipient name and I missed it so I'll add a test for that

Reply:
While we are on that subject I was trying to understand why do the question numbers differ between `InstructorFeedbackSubmitPageUiTest.json` and the actual page. I thought the test data is extracted from the json file so it should be exactly the same as what is displayed ?